src/schemas/22-0994/schema.js

```diff
@@ -0,0 +1,177 @@
+import _ from "lodash/fp";
+import { countries } from "../../common/constants";
+import definitions from "../../common/definitions";
+import {
+  documentTypes526,
+  pciuCountries,
+  pciuStates
+} from "../../common/constants";
+
+const baseAddressDef = {
```

Comment:
@imeecuison - I don't see a need to define address separately in this schema. We can re-use the definition of `address` itself similar to the other education forms.